dockerfiles/cli/scripts/cli.sh

```diff
@@ -115,37 +114,40 @@ cmd_init_reinit_pre_action() {
 cmd_start_check_ports() {
 
   # If dev mode is on, then we also need to check the debug port set by the user for availability
-  if [ "${CHE_DEVELOPMENT_MODE}" = "on" ]; then
+  if debug_server; then
     USER_DEBUG_PORT=$(docker_run --env-file="${REFERENCE_CONTAINER_ENVIRONMENT_FILE}" alpine sh -c 'echo $CODENVY_DEBUG_PORT')
 
     if [[ "$USER_DEBUG_PORT" = "" ]]; then
       # If the user has not set a debug port, then use the default
       CODENVY_DEBUG_PORT=8000
+      CHE_DEBUG_PORT=8000
```

Comment:
The CHE_DEBUG_PORT is what we inherited from how the Che CLI works.  However, in the codenvy.env file, we wanted users to set this as CODENVY_DEBUG_PORT.  So we need both here so that we can set the proper parameters from the inherited CLI, and for the user configuration here.